### PR TITLE
Feat/va checkbox docs

### PR DESCRIPTION
**Pull request by raichev-dima**
Should close https://github.com/epicmaxco/vuestic-ui/issues/318

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/components/page-configs/va-checkbox/api-options.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    arrayValue: {
+      local: true,
+    },
+    label: {
+      local: true,
+    },
+    leftLabel: {
+      local: true,
+    },
+    trueValue: {
```
Comment (asvae) [inline packages/docs/components/page-configs/va-checkbox/api-options.ts:17]: Any reason to keep those local? They're part of `SelectableMixin`.
For full list of related props - check said mixin.

File: packages/docs/components/page-configs/va-checkbox/api-options.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    checkedIcon: {
+      local: true,
+    },
+    indeterminateIcon: {
+      local: true,
+    },
+  },
+  events: {
+    input: {
+      types: '(value: any) => void',
```
Comment (asvae) [inline packages/docs/components/page-configs/va-checkbox/api-options.ts:35]: `Event` would make more sense here.

File: packages/docs/examples/va-checkbox/Array.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  data () {
+    return {
+      selection: ['one', 'four'],
+    }
+  },
+}
+</script>
+
+<style scoped>
+  .example {
```
Comment (asvae) [inline packages/docs/examples/va-checkbox/Array.vue:38]: Grid classes.

File: packages/docs/examples/va-checkbox/Error.vue [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <va-checkbox
+      v-model="value"
+      error
+      :error-messages="['Error message', 'Another error message', 'One more error message']"
```
Comment (asvae) [inline packages/docs/examples/va-checkbox/Error.vue:6]: No need to showcase multiple error messages, one is good. We'll likely cover these on validation page.

File: packages/docs/examples/va-checkbox/Label.vue [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      value: false,
+      label: 'Label',
+      longLabel:
+          'This label is pretty much long, but still perfectly fits the layout...',
```
Comment (asvae) [inline packages/docs/examples/va-checkbox/Label.vue:16]: pretty much long > pretty long

File: packages/docs/locales/en/en.json [hunk 6]
```diff
@@ -244,10 +244,27 @@
...
+    },
+    "VaCheckbox": {
+      "props": {
+        "value": "The value of the checkbox",
+        "arrayValue": "Same as native `value`. It is used with `v-model` of an array type",
+        "label": "Same as native `label`",
+        "leftLabel": "Moves the label on the left of a checkbox",
+        "trueValue": "Overrides the returned value when it's checked",
+        "falseValue": "Overrides the returned value when it's not checked",
+        "indeterminate": "Same as native `indeterminate`",
```
Comment (asvae) [inline packages/docs/locales/en/en.json:256]: Some details would be nice to have. We're not expecting all users to know html spec by heart.

**Comment by asvae** [comment]
Overall good. Couple of small fixes. Then merge.

File: packages/docs/components/page-configs/va-checkbox/api-options.ts [hunk 7]
```diff
@@ -0,0 +1,40 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```
Comment (asvae) [inline packages/docs/components/page-configs/va-checkbox/api-options.ts:1]: Not really sure why checkbox had flown to the right:
image

Here's the example from demo:
image

Comment (raichev-dima) [reply packages/docs/components/page-configs/va-checkbox/api-options.ts:1]: Hm... I'll check

**Pull request merged**